web/src/util/services/templates.ts

```diff
@@ -75,6 +75,10 @@ interface Manifest {
   INSTALLER_YAML: string;
 }
 
+function bashStringEscape( unescaped :string): string {
+  return unescaped.replace(/[!"'\\]/g, "\\\$&");
```

Comment:
I tried using the soln from https://qntm.org/bash, but it was escaping whitespace characters and unreadable. I also tried an expanded set of characters including $, but this also seemed like overkill and I'm not sure it would be the desired behavior.

Reply:
Would it be possible to get a test for this?

Reply:
(if we even have a test framework here...)

Reply:
You want it, you got :D 

Not doing a anthing crazy but at least there's a placeholder file now for folks to add service tests.